test/e2e/ctx/ctx.go

```diff
@@ -116,8 +117,7 @@ func (ctx TestContext) DumpNamespaceArtifacts(namespace string) error {
 		"KUBECONFIG=" + kubeconfigPath,
 	}
 
-	// compiled test binary running e2e tests is run from the root ./bin directory
-	cmd := exec.Command("../test/e2e/collect-ci-artifacts.sh")
```

Comment:
I ran into a sourcing issue when running the `make e2e` command locally as it's working directory points to ./test/e2e, so there's some skew between the working directory when using the kind or kubeconfig provisioners.